### ISSUE TITLE
Examples in section "for and iterators" fail

### ISSUE THREAD

**Issue by Element-126** [rust-lang/rust-by-example#1013]
Hi,

In section "for and iterators" (page 8.4), all three unmodified examples fail to compile, e.g.

```rust
let names = vec!["Bob", "Frank", "Ferris"];

for name in names.into_iter() {
    match name {
        "Ferris" => println!("There is a rustacean among us!"),
        _ => println!("Hello {}", name),
    }
}
```
```
   Compiling playground v0.0.1 (file:///playground)
error: expected item, found `let`
 --> src/main.rs:1:1
  |
1 | let names = vec!["Bob", "Frank", "Ferris"];
  | ^^^

error: aborting due to previous error

error: Could not compile `playground`.

To learn more, run the command again with --verbose.
```

This seems to have been introduced in #935.

**Comment by rgaertner** [comment]
This error occurs when statements are used without a function context. 
By wrapping these statements in a main function will fix it.

**Comment by Element-126** [comment]
Oops ! It was so trivial that I did not see it. PR incoming.

**Comment by rgaertner** [comment]
oh sorry, just created one

**Comment by QuietMisdreavus** [comment]
Closed in #1014 :grin:
